### ISSUE TITLE
Run/debug configuration for ROS master node

### ISSUE THREAD

**Issue by breandan** [duckietown/hatchery#2]
There is some documentation on [building a custom run configuration](https://www.jetbrains.org/intellij/sdk/docs/basics/run_configurations/run_configuration_management.html). Feel free to ping me if you have any questions.

**Comment by breandan** [comment]
For reference of a complete run configuration implementation, see the [Bash Support plugin](https://github.com/BashSupport/BashSupport/tree/master/src/com/ansorgit/plugins/bash/runner).

**Comment by machinekoder** [comment]
I like the idea of having run config for ROS launch and test files similar to Python/Bash execution:
![image](https://user-images.githubusercontent.com/1467368/43955474-b18d9964-9ca0-11e8-96fe-83970dbcc7c9.png)
`Run my_node.launch...`

**Comment by breandan** [comment]
Yep, thanks for the suggestion! We’re working on it, should be available in the next release.
